### PR TITLE
Update E2E instruction to clarify about the E2E workflows

### PR DESCRIPTION
**Pull request by tcnghia**
# Description

Update E2E doc to clarify about the E2E workflows

## Issue reference

Please reference the issue this PR will close: #N/A

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [x] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: tests/docs/running-e2e-test.md [hunk 1]
```diff
@@ -128,8 +128,15 @@ make delete-test-namespace
...
-
+To keep the build infrastructure simple, Dapr uses [dapr-test GitHub
+Actions
+Workflow](https://github.com/dapr/dapr/actions?query=workflow%3Adapr-test)
+to run e2e tests using one of [AKS
+clusters](https://github.com/dapr/dapr/blob/4cd61680a3129f729deae24a51da241d0701376c/tests/test-infra/find_cluster.sh#L12-L17). A
+separate workflow also runs E2E in [KinD](https://kind.sigs.k8s.io/)
+clusters.
+
+Once a contributor creates pull request, E2E tests on KinD clusters
```
Comment (yaron2) [inline tests/docs/running-e2e-test.md:139]:
```suggestion
Once a contributor creates a pull request, E2E tests on KinD clusters
```

File: tests/docs/running-e2e-test.md [hunk 2]
```diff
@@ -128,8 +128,15 @@ make delete-test-namespace
...
+are automatically executed for faster feedback. In order to run the
+E2E tests on AKS, ask a maintainer to add `/ok-to-test` comment to
```
Comment (yaron2) [inline tests/docs/running-e2e-test.md:141]:
```suggestion
E2E tests on AKS, ask a maintainer or approver to add `/ok-to-test` comment to
```

**Review by yaron2 — CHANGES_REQUESTED** [review]

**Comment by tcnghia** [comment]
@yaron2 feedbacks addressed. thanks!

**Review by yaron2 — APPROVED** [review]
lgtm

**Pull request merged**